### PR TITLE
Add metric semantic conventions for timed operations

### PR DESCRIPTION
**Pull request by justinfoote**
This OTEP attempts to address https://github.com/open-telemetry/opentelemetry-specification/issues/654.

I'd like us to come to agreement that we ought to have semantic conventions describing timing metrics.  These semantic conventions will describe what instrument(s) are created, how they are named, and what labels are applied to them.  They will not describe how or where the instruments are created and incremented.

### CODE REVIEW

**Comment by justinfoote** [comment]
@open-telemetry/specs-metrics-approvers would y'all have a look at this?  I'd like us to come to an agreement sooner rather than later that we want some semantic conventions about naming and labeling of metrics, and that we're not intending to specify API or SDK changes with them.

File: text/metrics/129-semantic-conventions-for-timed-operations.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+## Explanation
+
+Summarization of timed operations may be done with a single `ValueRecorder`, onto which
+the duration of each instance of the timed operation is recorded.
+
+### Naming
+
+The semantic conventions should specify the naming of this metric instrument, following
+the [Metric Instrument Naming Guidelines](./108-naming-guidelines.md#guidelines).
```
Comment (cijothomas) [inline text/metrics/129-semantic-conventions-for-timed-operations.md:18]: broken link?

Comment (justinfoote) [reply text/metrics/129-semantic-conventions-for-timed-operations.md]: 😬 Fixed.
Thanks!

**Comment by fatsheep9146** [comment]
Any update on this? 
@jmacd @MrAlias @justinfoote

**Comment by tedsuo** [comment]
@justinfoote we are marking OTEPs which are no longer being updated as `stale`. We will close this PR in one week. If you wish to resurrect this PR, that is fine.

**Pull request closed without merging**